.github/workflows/ci-static-build.yml

```diff
@@ -0,0 +1,24 @@
+name: "static-build-p4c"
```

Comment:
I find these files uncreadable. Can you add a little comment about what they provide?

Reply:
I added this one. I can make a separate pull request with a description. This one just tests for Ubuntu whether we can create a statically linked build of P4C.

Reply:
Reposting with the right acc. Yeah I added this workflow with the last pull request. The file shows up as new because it was renamed. Will add some more description in a new request. Essentially, it just compiles p4c with the BUILD_STATIC_RELEASE=ON option. To test whether a statically linked build works with Ubuntu 16.04.